main/views_test.py

```diff
@@ -41,25 +39,30 @@ def test_index_anonymous(settings, mocker, client):
     }
 
 
-@pytest.mark.django_db
-def test_index_logged_in(client):
-    """Verify the user is redirected to pay if logged in"""
-    user = UserFactory.create()
-    client.force_login(user)
-    assert client.get(reverse("bootcamp-index")).status_code == HTTP_302_FOUND
-
-
-@pytest.mark.django_db
-def test_index_logged_in_post(client):
-    """
-    Verify the user is redirected to the applications dashboard if logged in on a POST request
-    without CSRF token and that the GET parameters are kept
-    """
-    user = UserFactory.create()
-    client.force_login(user)
-    resp = client.post(reverse("bootcamp-index") + "?foo=bar")
-    assert resp.status_code == HTTP_302_FOUND
-    assert resp.url == reverse("applications") + "?foo=bar"
+# These tests need to be updated once it's confirmed the CMS HomePage should redirect to `/pay/`
```

Comment:
You can remove these tests. I don't think we will be redirecting to `/pay/` in the future.

Reply:
Removed